app.js

```diff
@@ -1,12 +1,26 @@
 const express = require('express')
 
-const { checkServerUp } = require('./1-controllers/server-controller')
+const { checkServerUp } = require('./1-controllers/api-controller')
+const { getTopics } = require('./1-controllers/topics-controller')
+const { handleCustomErrors, handlePsqlErrors, handleServerErrors, handle404 } = require('./error_handler')
+
+
+
 
 const app = express()
 
 
 app.get('/api', checkServerUp)
 
+app.get('/api/topics', getTopics)
+
+
+app.all('*', handle404)
+
+app.use(handlePsqlErrors)
+app.use(handleCustomErrors)
+app.use(handleServerErrors)
```

Comment:
Technically, this code should be here yet as you dont need it for this ticket. It would cause issue if another dev is working on the ticket that requires this code, and they also wrote it.

Reply:
They question why you're adding stuff not needed for your ticket.